### PR TITLE
fix(ui): license header added for missing files

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes :

I worked on the below changes

- license header added for missing files
- fix localisation keys

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/webpack.config.dev.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ module.exports = {
     proxy: [
       {
         context: '/api',
-        target: 'http://localhost:8585/',
+        target: 'https://sandbox.open-metadata.org/',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/webpack.config.dev.js:259]: Please revert this.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
@chirag-madlani , I would recommend scoping the PR with one type of change.

File: openmetadata-ui/src/main/resources/ui/webpack.config.dev.js (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/webpack.config.dev.js]: Done

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 2]
```diff
@@ -624,7 +623,7 @@
     "scopes-required": "Scopes is required",
     "secret-key-required": "SecretKey is required",
     "token-end-point-required": "TokenEndpoint is required",
-    "privateKey-required": "PrivateKey is required",
+    "private-key-required": "PrivateKey is required",
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:626]: Yes done updated

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 3]
```diff
@@ -498,7 +498,6 @@
     "manage-rule": "Manage Rule",
     "users": "Users",
     "user-permissions": "User Permissions",
-    "openmetadata": "OpenMetadata",
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:501]: Not needed anymore

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**